examples/seismic/skew_self_adjoint/ssa_01_iso_implementation1.ipynb

```diff
@@ -131,6 +129,37 @@
     "In the following notebooks in this series, material parameters *sandwiched* between the derivatives -- including anisotropy parameters -- become much more interesting, but here buoyancy $b$ is the only material parameter between derivatives in our skew self adjoint (SSA) wave equation. "
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Ensure matplotlib installed for cloud hosted jupyterhub servers"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: matplotlib in /cpfs/lfs03/ESDRD/wask/local/lib/python3.8/site-packages (3.2.1)\n",
```

Comment:
these are system-specific paths specific of your machine, so we probably don't want them here

Reply:
pretty sure that is output? I have #NBVAL_INGNORE_OUTPUT ignore at top of the cell.

Reply:
yeah but do we really want your paths (e.g. `../.. /wask/...`) `/cpfs/lfs03/ESDRD/wask/local/lib/python3.8/site-packages` in the cell output?

Reply:
OK, added ```%%capture``` at the top of the cell.